Formula/g/glib.rb

```diff
@@ -33,10 +33,6 @@ class Glib < Formula
     depends_on "gettext"
   end
 
-  on_system :linux, macos: :mojave_or_older do
-    depends_on "python-setuptools" => :build # for gobject-introspection
```

Comment:
any ref for this change?

Reply:
see above, it's already a general dependency.